<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>800</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>12</red>
        <green>12</green>
        <blue>12</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>10</red>
        <green>10</green>
        <blue>10</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>4</red>
        <green>4</green>
        <blue>4</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>5</red>
        <green>5</green>
        <blue>5</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>4</red>
        <green>4</green>
        <blue>4</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>12</red>
        <green>12</green>
        <blue>12</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>10</red>
        <green>10</green>
        <blue>10</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>4</red>
        <green>4</green>
        <blue>4</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>5</red>
        <green>5</green>
        <blue>5</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>4</red>
        <green>4</green>
        <blue>4</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>12</red>
        <green>12</green>
        <blue>12</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>10</red>
        <green>10</green>
        <blue>10</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>4</red>
        <green>4</green>
        <blue>4</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>5</red>
        <green>5</green>
        <blue>5</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>8</red>
        <green>8</green>
        <blue>8</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color : #080808;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>971</width>
      <height>651</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Pretendard</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #080808;

</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="autoScrollMargin">
     <number>0</number>
    </property>
    <property name="textElideMode">
     <enum>Qt::TextElideMode::ElideRight</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
     <property name="font">
      <font>
       <family>Pretendard</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
     <property name="font">
      <font>
       <family>Pretendard</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
     <property name="font">
      <font>
       <family>Pretendard</family>
      </font>
     </property>
    </column>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1251</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url('/home/llly/spirit/source/bg.png');
background-repeat: no-repeat; 
background-position: center;  
border: none;              
color:#ffffff;

background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="user_num">
    <property name="geometry">
     <rect>
      <x>1133</x>
      <y>12</y>
      <width>63</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pretendard</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transmission;
color:#A2A2A2;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1104</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background: #000000;
    color: white;
    border: 0.5px solid #383838;
    border-radius: 5px;
}

QComboBox QAbstractItemView {
    background-color:#181818; /* 배경 빨간색 */
    color: white;
    selection-background-color: #080808;
    selection-color: white;
    

    border:0.5px solid #383838; /* 테두리도 배경과 같은 색으로 */

}</string>
    </property>
    <item>
     <property name="text">
      <string>최신순</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>오래된 순</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>다운로드 순</string>
     </property>
    </item>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>181</width>
      <height>671</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pretendard</family>
     </font>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color:#ffffff;
</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
    </property>
    <property name="indentation">
     <number>10</number>
    </property>
    <property name="uniformRowHeights">
     <bool>true</bool>
    </property>
    <property name="itemsExpandable">
     <bool>true</bool>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Asset</string>
     </property>
     <property name="font">
      <font>
       <family>Pretendard Light</family>
      </font>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsEnabled</set>
     </property>
     <item>
      <property name="text">
       <string>3D Model</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Material</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HDRI</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Texture</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Category</string>
     </property>
     <property name="font">
      <font>
       <family>Pretendard Light</family>
      </font>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsEnabled</set>
     </property>
     <item>
      <property name="text">
       <string>Character</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Environment</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Prop</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vehicle</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Weapon</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Architecture</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Others</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Style</string>
     </property>
     <property name="font">
      <font>
       <family>Pretendard Light</family>
      </font>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsEnabled</set>
     </property>
     <item>
      <property name="text">
       <string>Realistic</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stylized</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Procedural</string>
      </property>
      <property name="font">
       <font>
        <family>Pretendard Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QLabel" name="toggle_btn">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>67</y>
      <width>115</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transmission;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle_btn_touch_area">
    <property name="geometry">
     <rect>
      <x>949</x>
      <y>64</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: transparent;  /* 배경 투명 */
    border: none;  /* 테두리 제거 */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="search">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>72</y>
      <width>631</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pretendard</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>471</width>
      <height>851</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="exit_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
color:#ffffff;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4"/>
   </widget>
   <zorder>label</zorder>
   <zorder>user_num</zorder>
   <zorder>tableWidget</zorder>
   <zorder>comboBox</zorder>
   <zorder>treeWidget</zorder>
   <zorder>toggle_btn</zorder>
   <zorder>toggle_btn_touch_area</zorder>
   <zorder>search</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
